### PR TITLE
Add framer-motion extension

### PR DESCRIPTION
**Pull request by madebyankur**
## Description

Adding an extension to link to Framer Motion docs (https://www.framer.com/motion/) and a glossary of the utilities available to copy directly.

## Screencast

https://github.com/raycast/extensions/assets/12908185/e7ae6498-33e5-45a4-a79b-15e65c1b5638

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/framer-motion/src/utilities.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      {UTILITY_ITEMS.map((items) => (
+        <List.Section key={items.section} title={items.section}>
+          {items.items.map((item) => (
+            <List.Item
+              key={item.title + Math.random()}
+              title={item.title}
+              subtitle={item.subtitle}
+              actions={
+                <ActionPanel>
+                  <Action.CopyToClipboard title={item.title} content={item.content} />
```
Comment (samuelkraft) [inline extensions/framer-motion/src/utilities.tsx:17]: It's unclear what this action does

```suggestion
                  <Action.CopyToClipboard title="Copy Utility" content={item.content} />
```

File: extensions/framer-motion/src/utilities.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+export default function Command() {
+  return (
+    <List>
+      {UTILITY_ITEMS.map((items) => (
+        <List.Section key={items.section} title={items.section}>
+          {items.items.map((item) => (
+            <List.Item
+              key={item.title + Math.random()}
+              title={item.title}
+              subtitle={item.subtitle}
```
Comment (samuelkraft) [inline extensions/framer-motion/src/utilities.tsx:14]: Adding the section title as a keyword for each item could be useful if you're not sure what the utility is named e.g. searching for just "3d" or "transition".

Sam 2024-03-22 at 10  23 49@2x



```suggestion
              subtitle={item.subtitle}
              keywords={[items.section]}
```

Comment (samuelkraft) [inline extensions/framer-motion/src/utilities.tsx:14]: Adding the section title as a keyword for each item could be useful if you're not sure what the utility is named e.g. searching for just "3d" or "transition".

Sam 2024-03-22 at 10  23 49@2x



```suggestion
              subtitle={item.subtitle}
              keywords={[items.section]}
```

File: extensions/framer-motion/package.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  "author": "madebyankur",
+  "categories": [
+    "Developer Tools",
+    "Design Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Search Documentation",
```
Comment (samuelkraft) [inline extensions/framer-motion/package.json:16]:
```suggestion
      "title": "Search Documentation",
      "subtitle": "Framer Motion",
```

File: extensions/framer-motion/package.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+      "description": "Search Framer Motion documentation",
+      "mode": "view"
+    },
+    {
+      "name": "utilities",
+      "title": "Search Utilities",
```
Comment (samuelkraft) [inline extensions/framer-motion/package.json:22]:
```suggestion
      "title": "Search Utilities",
      "subtitle": "Framer Motion",
```

**Review by samuelkraft — CHANGES_REQUESTED** [review]
Hey @madebyankur, thanks for the contribution!

I've reviewed your extension and have some minor comments and suggestions:

* Please remove the metadata-1 image since it does not add much contextual value, metadata-2 shows the extension much better 🙂  

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by madebyankur** [comment]
@samuelkraft thanks for the review 🙌

I'll try to get these changes pushed in.

**Review by samuelkraft — APPROVED** [review]
Thanks for the changes @madebyankur, love the icon additions!

**Pull request merged**